client/state/page/reducer.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	PAGE_LINK_ADD,
+	PAGE_META_ADD,
+	PAGE_TITLE_SET,
+	PAGE_UNREAD_COUNT_SET
+} from 'state/action-types';
+
+export function title( state = '', action ) {
```

Comment:
It hadn't come to mind 'til now, but we should consider [data persistence](https://github.com/Automattic/wp-calypso/blob/master/docs/our-approach-to-data.md#data-persistence--2754-).

By default, we store state to IndexedDB any time a change occurs and restore it on the next page session. This is unless you opt out by specifying handlers for the `SERIALIZE` and `DESERIALIZE` action types.

I _think_ we'd be okay since controllers using legacy `screen-title` actions would always be run _after_ the state has been restored. That said, it seems like something we should not be persisting, so we should probably save ourselves the potential trouble. See ["Not Persisting Data"](https://github.com/Automattic/wp-calypso/blob/master/docs/our-approach-to-data.md#not-persisting-data) as reference.